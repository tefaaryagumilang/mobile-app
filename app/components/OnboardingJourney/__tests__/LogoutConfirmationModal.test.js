import React from 'react';
import renderer from 'react-test-renderer';
jest.mock('lodash');

import LogoutConfirmationModal from '../LogoutConfirmationModal.component';

describe('LogoutConfirmationModal component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<LogoutConfirmationModal />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
