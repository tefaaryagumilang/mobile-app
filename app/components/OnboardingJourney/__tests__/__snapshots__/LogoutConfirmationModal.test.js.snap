// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoutConfirmationModal component renders correctly 1`] = `
<Modal
  animationType="none"
  hardwareAccelerated={false}
  hideOnBack={true}
  onBackButtonPress={[Function]}
  onRequestClose={[Function]}
  transparent={true}
  visible={false}
>
  <View
    style={
      Object {
        "backgroundColor": "transparent",
        "bottom": 0,
        "height": 1334,
        "left": 0,
        "opacity": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
        "width": 750,
      }
    }
  />
  <View
    hideOnBack={true}
    onBackButtonPress={[Function]}
    style={
      Object {
        "flex": 1,
        "justifyContent": "center",
        "margin": 0,
        "padding": 0,
      }
    }
    transparent={true}
  >
    <View
      contentContainerStyle={
        Object {
          "flex": 1,
        }
      }
      extraHeight={120}
      keyboardShouldPersistTaps="handled"
      scrollEnabled={false}
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        accessible={true}
        clickable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "rgba(37, 8, 10, 0.78)",
              "flex": 1,
              "justifyContent": "center",
              "padding": 20,
            },
            Object {},
          ]
        }
      >
        <View
          accessible={true}
          clickable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#FFFFFF",
                "borderRadius": 20,
                "padding": 20,
              },
              Object {},
              Object {
                "alignItems": "center",
                "backgroundColor": "#FFFFFF",
                "borderRadius": 20,
                "padding": 20,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000",
                  "fontSize": 24,
                },
                Object {
                  "textAlign": "center",
                },
                Object {
                  "color": "#000",
                  "paddingTop": 10,
                },
              ]
            }
          >
            Log Out?
          </Text>
          <Text
            style={
              Array [
                Object {
                  "textAlign": "center",
                },
                Object {
                  "paddingBottom": 30,
                  "paddingHorizontal": 20,
                  "paddingTop": 20,
                },
              ]
            }
          >
            Apakah Anda ingin logout?
          </Text>
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "flex": 1,
                }
              }
            >
              <View
                accessible={true}
                clickable={true}
                isTVSelectable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "opacity": 1,
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#000",
                        "fontSize": 16,
                      },
                      Object {
                        "color": "#666",
                      },
                    ]
                  }
                >
                  BATAL
                </Text>
              </View>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "flex": 1,
                }
              }
            >
              <View
                accessible={true}
                clickable={true}
                isTVSelectable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "opacity": 1,
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#000",
                        "fontSize": 16,
                      },
                      Object {
                        "color": "#FA3F70",
                      },
                    ]
                  }
                >
                  Ya, LOG OUT
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</Modal>
`;
